### ISSUE TITLE
Do we have any Command line for Gradle as well? like we have it for maven (like using with -Drewrite)

### ISSUE THREAD

**Issue by Sanskarbajaj** [openrewrite/rewrite-gradle-plugin#263]
Do we have any Command line for Gradle as well? like we have it for maven (like using with -Drewrite)

**Comment by shanman190** [comment]
Hi, @Sanskarbajaj! Would you be able to explain a bit more what feature of the Maven plugin that you're particularly interested in? For the most part, the two plugins have feature parity with each other.

**Comment by timtebeek** [comment]
Hi @Sanskarbajaj ; Could it be that you're referring to the ability to run OpenRewrite against Maven projects without changing the pom.xml build file?
https://docs.openrewrite.org/running-recipes/running-rewrite-on-a-maven-project-without-modifying-the-build

We have a similar page in the docs with an approach for Gradle; found at
https://docs.openrewrite.org/running-recipes/running-rewrite-on-a-gradle-project-without-modifying-the-build

**Comment by Sanskarbajaj** [comment]
yes @timtebeek I'm referring to the same link for Gradle. but not able to run it. Do you have a dummy sample full-fledged sample to refer through?. coz for maven we're able to run everything we want and need help in gradle one :)

**Comment by timtebeek** [comment]
Could you share what steps you've tried and what you encountered? Perhaps that might help us update the docs.
